frontend/public/components/resource-list.tsx

```diff
@@ -113,7 +113,9 @@ export const ResourceDetailsPage = connectToPlural((props: ResourceDetailsPagePr
   return (
     <>
       <Helmet>
-        <title>{`${decodedName} · Details`}</title>
+        <title
+          data-title-id={`${props.match.params.name} · ${props.kindObj?.kind}`}
+        >{`${decodedName} · Details`}</title>
```

Comment:
Hey, can you please:

1. Skip the resource name (pod-1234) from telemetry.
2. And can we add the translated resource kind (incl. the Helm utils you have mentioned) for the user here as well?

Thanks :)

```suggestion
        <title
          data-title-id={props.kindObj?.kind}
        >{`${decodedName} · ${translatedResourceKind} · Details`}</title>
```